### DISCUSSION TITLE
How can I  git students repo URL on Github classroom ?

### DISCUSSION THREAD

**Discussion by sian-0018** [community/community#20109] [GitHub Education] [answered]
Generally speaking, in Github, you can use code to get the time, information and author of commit and push, but in github classroom, how to capture the commit and push information of all students in the classroom from the perspective of the teacher?

**Comment by Zi-Tao** [comment, marked as answer]
@sian-0018 
https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-an-individual-assignment#monitoring-students-progress
When a student finishes an assignment, you can review the files in the repository, or you can review the history and visualizations for the repository to better understand the student's work. For more information, see "[Visualizing repository data with graphs](https://docs.github.com/en/github/visualizing-repository-data-with-graphs)."

**Reply by sian-0018** [reply]
Thank you for your answer

**Reply by sian-0018** [reply]
What is the way to get the data to display the records of all students in a row format such as in .json?

**Reply by Zi-Tao** [reply]
@sian-0018 
https://education.github.community/t/new-collection-of-tools-to-manage-github-classrooms/41450
https://github.com/danwallach/github-classroom-utils
